addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -223,6 +254,49 @@ func (r *MirrorPeerReconciler) enableCSIAddons(ctx context.Context, namespace st
 	return nil
 }
 
+func (r *MirrorPeerReconciler) fillClusterFSIDs(ctx context.Context, mp *multiclusterv1alpha1.MirrorPeer) []error {
```

Comment:
update it to return the FSIDs instead of string it in some magic variable.

also, no need to return array of error. If there is an error in getting any FSID, whole process will fail. So no need to keep on trying after first failure, just return first error.

Reply:
I forgot to remove it. Was using it for testing during whole secret fetching part.